atd-vzv/src/views/summary/DemographicsByYear.js

```diff
@@ -0,0 +1,312 @@
+                record.prsn_age < 18 && accumulator++;
+                break;
+              case 2:
+                record.prsn_age >= 18 && record.prsn_age <= 44 && accumulator++;
+                break;
+              case 3:
+                record.prsn_age > 44 && record.prsn_age <= 64 && accumulator++;
+                break;
+              case 4:
+                record.prsn_age > 64 && accumulator++;
+                break;
+              case "noData":
+                !record.prsn_age && accumulator++;
+                break;
+              default:
+                break;
+            }
+            break;
+          case "prsn_gndr_id":
+            record.prsn_gndr_id === `${categoryValue}` && accumulator++;
+            break;
+          case "prsn_ethnicity_id":
+            // If the ethnicity id value matches the category value, increment the count for the associated category
+            record.prsn_ethnicity_id === `${categoryValue}` && accumulator++;
+            // If the ethnicity id value is missing and the category value is "noData", increment the count for the "No data" category
+            !record.prsn_ethnicity_id &&
+              categoryValue === "noData" &&
+              accumulator++;
+            break;
+          default:
+            break;
+        }
+        return accumulator;
+      }, 0);
+      const percentage = (categoryTotal / chartData[year].length) * 100;
+      return percentage;
+    });
+
+  // Sort category order in stack by averaging total demographic stats across all years in chart
+  const sortData = data => {
+    const averageCrashes = dataArray =>
+      dataArray.reduce((a, b) => a + b) / dataArray.length;
+    return data.sort((a, b) => averageCrashes(b.data) - averageCrashes(a.data));
+  };
+
+  // Create dataset for each demographic type
+  const createTypeDatasets = () => {
+    let categories;
+    switch (activeTab) {
+      case "prsn_age":
+        categories = ageCategories;
+        break;
+      case "prsn_gndr_id":
+        categories = sexCategories;
+        break;
+      case "prsn_ethnicity_id":
+        categories = raceCategories;
+        break;
+      default:
+        break;
+    }
+    const data = categories.map(category => ({
+      backgroundColor: category.color,
+      borderColor: category.color,
+      borderWidth: 2,
+      hoverBackgroundColor: category.color,
+      hoverBorderColor: category.color,
+      label: category.label,
+      data: getData(category.categoryValue)
+    }));
+    // If age is selected, keep original sorting to make chart more readable
+    // For other categories, determine order of category (highest to lowest proportion)
+    if (activeTab === "prsn_age") {
+      return data;
+    } else {
+      return sortData(data);
+    }
+  };
+
+  const data = {
+    labels: createChartLabels(),
+    datasets: !!chartData && createTypeDatasets()
+  };
+
+  return (
+    <Container>
+      <Row style={{ paddingBottom: "0.75em" }}>
```

Comment:
instead of hardcoding a random fixed value, could you use a bootstrap utility class like `className="pb-3"`
https://getbootstrap.com/docs/4.4/utilities/spacing/

This will ensure consistency over time.